doc/contributing/maintaining-single-executable-application-support.md

```diff
@@ -0,0 +1,81 @@
+# Maintaining Single Executable Applications support
+
+Support for [single executable applications][] is one of the key technical
+priorities identified for the success of Node.js.
+
```

Comment:
Don't really think this bit is necessary to say. Non-blocking tho

Reply:
I saw this in a couple of other docs too like this one - https://github.com/nodejs/node/blob/a37c083dc138bb648be9a3581d1b2fe3462d4c8b/doc/contributing/maintaining-web-assembly.md?plain=1#L3-L6. Maybe all of these could be removed in a separate PR later on?